CRM/Contribute/Exception/FutureContributionPageException.php

```diff
@@ -8,7 +8,7 @@ class CRM_Contribute_Exception_FutureContributionPageException extends Exception
    * @param int $id
    */
   public function __construct($message, $id) {
-    parent::__construct(ts($message));
+    parent::__construct($message);
```

Comment:
I'm not sure what the intention here was, or what it may break, but I would rather see the bug surface so that we can fix the source.